test/regressions/index.js

```diff
@@ -254,7 +256,7 @@ function excludeDemoFixture(suite, name) {
     }
 
     // assume regex
-    if (pattern.test(suite)) {
+    if (pattern.test(suite) || pattern.test(`${suite}/${name}.png`)) {
```

Comment:
Taken from https://github.com/mui/mui-x/blob/ac189d75b5ee607e6f0e76e49b79c4f6472ec526/test/regressions/index.js#L48

Reply:
Looking at the logic, I think that this works, it will make it easier to unify the logic between the repositories in the future. 

```suggestion
    if pattern.test(`${suite}/${name}.png`)) {
```

Reply:
Applied 👍  I wasn't sure if the current logic was like that for some reason.